tools/ui-components/src/image/image.types.ts

```diff
@@ -1,7 +1,5 @@
 import React from 'react';
-export interface ImageProps {
+export interface ImageProps extends React.ImgHTMLAttributes<HTMLImageElement> {
   alt: string;
-  className?: string;
   src: string;
-  ref: React.RefObject<HTMLImageElement>;
```

Comment:
If I'm reading the errors right, it seems to think we still need this.

Reply:
we can stop using forwardRef and treat it as component with a ref, but forwardRef isn't part of the props and should be treated as its own argument.